README.md

```diff
@@ -76,8 +76,9 @@ Import { Chat } from 'botframework-webchat';
 
 const YourApp = () => {
     <div>
+        <link href="https://unpkg.com/botframework-webchat/botchat.css" rel="stylesheet" />
```

Comment:
There should be a file in your repo: **/node_modules/botframework-webchat/botchat.css**
It's up to you how to copy that file into your build process so that it gets shipped in your application.
Also, if there was a `<link>` tag, it should go in the head of your html document, and not in a `<div>`.